### ISSUE TITLE
multiple Workouts upload

### ISSUE THREAD

**Issue by effectpears** [dvmarinoff/Auuki#215]
Unfortunately, I regularly lose my settings, e.g. after an update. It would be great if there was an option to upload multiple workouts at the same time.

**Comment by dvmarinoff** [comment]
Yes, I guess that's what happened after the domain name change from `flux-web.vercel.app` to `auuki.com`. Although it shouldn't happen more than ones for every device you are using. Anyway the change has been necessary so I can bring the API features to production (uploading to strava and intervals.icu, fetching daily workouts from intervals.icu).

Anyway It's a good thing to have the multi file upload so I added this feature:

https://github.com/user-attachments/assets/9831fd97-10dd-4414-b398-3b11d9d912a8
